moped-database/migrations/1654057324421_alter_table_public_moped_project_files_alter_column_is_retired/down.sql

```diff
@@ -0,0 +1,2 @@
+COMMENT ON COLUMN "public"."moped_project_files"."is_retired" IS E'';
```

Comment:
TIL about E

Reply:
Same! Hasura auto-generated this so I had to look it up myself.